src/styles/Panel.scss

```diff
@@ -3,9 +3,9 @@
   bottom: calc(100%);
```

Comment:
doesn't look like the styles in this file are applied anywhere so we should delete